src/megatron/bridge/models/hf_pretrained/wan.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
this is too model specific, cannot be put here. move it to models/wan/hf_wrappers.py for example